src/features/click-to-play.js

```diff
@@ -1,13 +1,19 @@
 import { createCustomEvent, sendMessage, OriginalCustomEvent, originalWindowDispatchEvent } from '../utils.js'
 import { logoImg, loadingImages, closeIcon } from './click-to-play/ctl-assets.js'
-import { config, styles } from './click-to-play/ctl-config.js'
+import { styles, getConfig } from './click-to-play/ctl-config.js'
 
 let devMode = false
 let isYoutubePreviewsEnabled = false
 let appID
 
 const titleID = 'DuckDuckGoPrivacyEssentialsCTLElementTitle'
 
+/*********************************************************
+ *  config set by getConfig() at init per localization
+ *********************************************************/
+let config = {}
```

Comment:
Could we have these initialised to null?

Reply:
Sure tho we are mixing initializers (the next two are empty array & obj)